docs/source/example_notebooks/dowhy_causal_discovery_example.ipynb

```diff
@@ -487,4 +487,4 @@
  },
  "nbformat": 4,
```

Comment:
If possible, can we remove these 4 notebooks from the PR? The only change is a newline removal.